src/components/vote-form.js

```diff
@@ -13,45 +13,42 @@ function VoteForm() {
 
 	const getCandidates = async () => {
 		await axios
```

Comment:
```suggestion
	const getCandidates = axios
```
이렇게 바꿔줘도 의미가 같습니다! axios가 비동기 함수이기 때문이죠